### DISCUSSION TITLE
Gitignore

### DISCUSSION THREAD

**Discussion by baileys-am** [srsran/srsRAN_Project#761] [Ideas & Suggestions]
Is there a reason no gitignore has been added? Would love for the build folder to be ignored.

**Comment by frankist** [comment]
I generally create a `.gitignore` file that contains `.gitignore` itself as one of the files to be ignored. This way I can add to my `.gitignore` all sorts of build folder names and IDE/editor cache files to my srsRAN project directory without imposing those same folder/file names on others.
